notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -179,7 +179,7 @@
         "! (pip3 install --upgrade $USER_FLAG \\\n",
```

Comment:
<p>replace 'now' with 'UUID'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/985/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Replaced with generate_uuid() since we want a unique id if the cell is rerun.</p>